rosbag2_storage_mcap/test/rosbag2_storage_mcap/test_mcap_storage.cpp

```diff
@@ -200,3 +203,41 @@ TEST_F(TemporaryDirectoryFixture, can_write_mcap_with_zstd_configured_from_yaml)
   }
 }
 #endif  // #ifdef ROSBAG2_STORAGE_MCAP_HAS_STORAGE_OPTIONS
+
+TEST_F(TemporaryDirectoryFixture, mcap_contains_ros_distro)
+{
+  // Guarantee env var set for mcap to use - in a full-build testing environment it may not be.
+  const std::string current_ros_distro = "rolling";
```

Comment:
```suggestion
  const std::string current_ros_distro = "iron";
```

Reply:
This is a testing value and I want to just backport it verbatim - it's a process-scoped environment variable and therefore the actual value doesn't matter.

Reply:
Ok, never mind in this case.